### DISCUSSION TITLE
Getting email of user after Linkedin log in?

### DISCUSSION THREAD

**Discussion by mattfranciswork0** [nextauthjs/next-auth#2266] [Help] [answered]
It looks like with Linkedin, we are only able to get the full name and last name. 
```
export default function Index({ providers }) {
    const [session, loading] = useSession();
    console.log(session);
```

```
export default NextAuth({
    // Configure one or more authentication providers
        Providers.LinkedIn({
            clientId: process.env.LINKEDIN_ID,
            clientSecret: process.env.LINKEDIN_SECRET,
        }),
        // ...add more providers here
    ],](url)
```

Gives me
![image](https://user-images.githubusercontent.com/22691408/123571639-a77e3880-d798-11eb-82c8-43b4c1be5276.png)

Is it possible to get the email too? I was thinking of using the email received from useSession() as a way to identify different users in my database.

I believe the default provider does not return the email, as covered here:
https://github.com/nextauthjs/next-auth/discussions/1541

What can be done so that it **does** return the email?

**Comment by balazsorban44** [comment, marked as answer]
https://github.com/nextauthjs/next-auth/discussions/1541#discussioncomment-931773
